### DISCUSSION TITLE
Overture Maps Foundation membership

### DISCUSSION THREAD

**Discussion by EwoutH** [geopandas/geopandas#2722] [Ideas]
Most of you will probably have heard about this big splash today: [Linux Foundation Announces Overture Maps Foundation to Build Interoperable Open Map Data](https://www.linuxfoundation.org/press/linux-foundation-announces-overture-maps-foundation-to-build-interoperable-open-map-data).

They have a free [membership tier](https://overturemaps.org/become-a-member/) for nonprofits. It might be interesting to become a member (as part of NumFOCUS).

Since it falls under the Linux Foundation, I'm very curious where this is going!

**Comment by martinfleis** [comment]
Hmm, maybe together with Pangeo folks under the NumFOCUS umbrella. But I'm not sure how useful would it be for the geopandas project.
